pyproject.toml

```diff
@@ -92,3 +93,25 @@ module = [
   "traceloggingdynamic",
 ]
 ignore_missing_imports = true
+
+[tool.poe.tasks.install]
```

Comment:
@mshafer-NI I would appreciate your feedback on these `poethepoet` tasks.

If this seems like a good practice, we could start using it in our GitHub workflows.

Reply:
This does seem like a good practice, it is on my backlog to experiment/adopt as standard tooling.